cou/commands.py

```diff
@@ -316,13 +319,32 @@ def create_subparsers(parser: argparse.ArgumentParser) -> argparse._SubParsersAc
     return subparsers
 
 
-def parse_args(args: Any) -> argparse.Namespace:  # pylint: disable=inconsistent-return-statements
+class Namespace(NamedTuple):
```

Comment:
The name is strange, I will rename it to something like  `CLIargs`.

Reply:
or `COUargs`

Reply:
Now I realized that this also could be defined as frozen dataclass, lake that it will not be possible to change any arguments in any step accidentally.